### DISCUSSION TITLE
oh-my-posh won't load in bash even if I place the command for invoking it in `$HOME/.bash_profile`

### DISCUSSION THREAD

**Discussion by kris3713** [JanDeDobbeleer/oh-my-posh#4053] [Troubleshoot] [answered]
The title literally says it all. Also for context, I'm doing this on an Ubuntu Linux (Non-LTS) virtual machine.

Because it won't automatically load when I start a terminal session, I am forced to manually invoke it using this bash snippet:
```bash
eval "$(oh-my-posh init 'bash' --config "$(brew --prefix oh-my-posh)/themes/kushal.omp.json")"
```

Having to invoke this every time I start a bash terminal session just to make it look better is very repetitive. Is there something I am doing wrong? What is the correct way of putting the bash snippet in my profile file?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Zliced13 is the file loaded by bash when starting the terminal? Because if the command works when executing directly from the terminal, it should also work when added to a profile (or bashrc) file that's sourced by bash on startup.

**Reply by kris3713** [reply]
I figured it out, if I put the bash snippet in that in the `.bashrc` file, bash will read it. Now it works for me now! Thank you for helping me solve my problem, @JanDeDobbeleer!
